llvm/test/CodeGen/AIE/aie2/schedule/memory_access_DM_TM.mir

```diff
@@ -0,0 +1,89 @@
+
+# This test checks scheduling of LDA_TM with VLDB
+
+---
+name:            LDA_TM_VLDB_from_BankA
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: LDA_TM_VLDB_from_BankA
+    ; CHECK: BUNDLE implicit-def $r0, implicit-def $wl2, implicit killed $p0 {
+    ; CHECK-NEXT:   $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory", addrspace 15)
+    ; CHECK-NEXT:   $wl2 = VLDB_dmw_ldb_ag_idx_imm killed $p0, 0 :: (load (<8 x s32>), addrspace 5)
+    ; CHECK-NEXT: }
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory")
+    $wl2 = VLDB_dmw_ldb_ag_idx_imm $p0, 0 :: (load (<8 x s32>), addrspace 5)
+...
+
+---
+name:            LDA_TM_VLDB_from_BankB
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: LDA_TM_VLDB_from_BankB
+    ; CHECK: BUNDLE implicit-def $r0, implicit-def $wl2, implicit killed $p0 {
+    ; CHECK-NEXT:   $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory", addrspace 15)
+    ; CHECK-NEXT:   $wl2 = VLDB_dmw_ldb_ag_idx_imm killed $p0, 0 :: (load (<8 x s32>), addrspace 6)
+    ; CHECK-NEXT: }
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory")
+    $wl2 = VLDB_dmw_ldb_ag_idx_imm $p0, 0 :: (load (<8 x s32>), addrspace 6)
+...
+
+---
+name:            LDA_TM_VLDB_from_BankC
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: LDA_TM_VLDB_from_BankC
+    ; CHECK: BUNDLE implicit-def $r0, implicit-def $wl2, implicit killed $p0 {
+    ; CHECK-NEXT:   $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory", addrspace 15)
+    ; CHECK-NEXT:   $wl2 = VLDB_dmw_ldb_ag_idx_imm killed $p0, 0 :: (load (<8 x s32>), addrspace 7)
+    ; CHECK-NEXT: }
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory")
+    $wl2 = VLDB_dmw_ldb_ag_idx_imm $p0, 0 :: (load (<8 x s32>), addrspace 7)
+...
+
+---
+name:            LDA_TM_VLDB_from_BankD
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: LDA_TM_VLDB_from_BankD
+    ; CHECK: BUNDLE implicit-def $r0, implicit-def $wl2, implicit killed $p0 {
+    ; CHECK-NEXT:   $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory", addrspace 15)
+    ; CHECK-NEXT:   $wl2 = VLDB_dmw_ldb_ag_idx_imm killed $p0, 0 :: (load (<8 x s32>), addrspace 8)
+    ; CHECK-NEXT: }
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    $r0 = LDA_TM $p0 :: (load (s32) from custom "TileMemory")
+    $wl2 = VLDB_dmw_ldb_ag_idx_imm $p0, 0 :: (load (<8 x s32>), addrspace 8)
+...
```

Comment:
See https://github.com/Xilinx/llvm-aie/pull/95#discussion_r1713401640
Could you add a test where VLDB has no MMO? I think it should also get bundled. Maybe you can do all the changes suggested [here](https://github.com/Xilinx/llvm-aie/pull/95#discussion_r1713409406) in this commit actually.